html/changelogs/thegoret-fix-theta.yml

```diff
@@ -0,0 +1,41 @@
+################################
+# Example Changelog File
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read. If you change this file, you will look really dumb.
+#
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes: 
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   maptweak
+#   spellcheck (typo fixes)
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.  
+author: Goret
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - bugfix: "Thetamycin now metabolise at 0.05u per tick, as it used to be."
```

Comment:
This is wrong. 0.05 * REM means 0.05 * 0.02, which is the actual amount of reagents metabolized in a tick. This would mean 0.001u/t, which is less than the default, aka you made it worse.

Reply:
What? REM is 0.2 and this PR makes it 0.01u/t (Btw will need to edit the changelog still)

Metabolizing 0.01u/tick is slower than 0.2u/tick
The default metabolizing rate for reagents is REM. 
It's not worse, the CL is just inaccurate. This is what it was before MDP removed it in a brainmed commit.

Reply:
Though looking at it, it does require dose to be above 1 for the CE_ANTIBIOTIC effect to do anything.
This is way more fucked than initially thought.
Because we want it to start working fast, but linger longer.
`/datum/reagent/thetamycin/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
	M.add_chemical_effect(CE_EMETIC, dose/8) // chance per 2 second tick to cause vomiting
	M.add_chemical_effect(CE_ANTIBIOTIC, dose) // strength of antibiotics; amount absorbed, need >5 to be effective. takes 50 seconds to work
`
Idk what the best way to address this is.

Reply:
im retarded just change the cl and itll be fine

Reply:
No, wait, I'm dumb. That comment for the add_chemical_effect() is accurate to the rate goret adds in this. that's fine then

Reply:
all good